app/components/avo/fields/location_field/show_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= field_wrapper **field_wrapper_args do %>
   <% if field.value_present? %>
-    <%= js_map [{latitude: field.value[0], longitude: field.value[1]}] %>
+    <%= js_map [{latitude: field.value[0], longitude: field.value[1]}], **helpers.resource_mapkick_options(@resource.map) %>
```

Comment:
I added it because I thought it made sense for consistency, but now that you point it out that argument seems pretty week. I will remove that change. :+1: